app/models/state_file_md_intake.rb

```diff
@@ -265,7 +265,11 @@ def has_banking_information_in_financial_resolution?
     true
   end
 
-  def filer_disabled?
+  def has_at_least_one_disabled_filer?
     primary_disabled_yes? || spouse_disabled_yes?
   end
+
+  def should_warn_about_pension_exclusion?
```

Comment:
you’ve done so much work on this & i don’t want to make you go crazy, but maybe this should be `can_qualify_for_pension_exclusion?` 

i *hope* this isn’t too wild of a change & can easily be done in RubyMine.

Reply:
https://github.com/codeforamerica/vita-min/pull/5709#discussion_r1985846137

Arin didn't like that name since this could be false but a user could still qualify. What do you think?